pixel-point/pixelpoint-website
# chore: update Isovalent case study (add quote)

**Describe what changes this pull request brings**
This pull request adds update to Isovalent case study (add quote).

**Steps to test**

1. Open the [page](https://pixelpointwebsite-isovalentcasestudyupdate.gtsb.io/case-studies/)
2. Confirm that everything looks and works as expected

**Screenshots**
Isovalent case study
<details>
<img width="2010" alt="Screenshot 2022-09-15 at 14 24 36" src="https://user-images.githubusercontent.com/75138180/190370096-b01bfa7e-e477-4864-b648-6f04c752c27b.png">


<img width="2029" alt="Screenshot 2022-09-15 at 14 29 47" src="https://user-images.githubusercontent.com/75138180/190370102-736d41b4-2479-474b-8a84-bd8ab60162db.png">
</details>
